src/xrGame/ui/UIMapList.cpp

```diff
@@ -240,6 +240,8 @@ void CUIMapList::LoadMapList()
 {
     const auto& weathers = gMapListHelper.GetGameWeathers();
     u32 cnt = 0;
+    if(m_pWeatherSelector->GetSize() > 0);
```

Comment:
Please send it in another pull request, but also fix the missing space:
```suggestion
    if (m_pWeatherSelector->GetSize() > 0);
```